packages/logs/src/boot/logs.ts

```diff
@@ -153,15 +153,6 @@ export function buildAssemble(
   }
 }
 
-export function assembleMessageContexts(
```

Comment:
This function has no reference. So I've removed it instead of just fixing the camelcase naming.